adafruit_hue.py

```diff
@@ -161,20 +169,22 @@ def toggle_light(self, light_id):
         resp = self.set_light(light_id, on=light_state)
         return resp
 
-    def get_light(self, light_id):
+    def get_light(self, light_id: Union[int, str]) -> str:
         """Gets the attributes and state of a provided light.
         :param int light_id: Light identifier.
         """
-        resp = self._get("{0}/lights/{1}".format(self._username_url, light_id))
+        resp = self._get(f"{self._username_url}/lights/{light_id}")
         return resp
 
-    def get_lights(self):
+    def get_lights(self) -> str:
         """Returns all the light resources available for a bridge."""
         resp = self._get(self._username_url + "/lights")
         return resp
 
     # Groups API
-    def create_group(self, lights, group_id):
+    def create_group(
+        self, lights: List[Union[int, str]], group_id: Optional[str]
```

Comment:
I think based on the API reference for Hue V1, this can be any string of length 0 to 32.  So the optional part probably means you can just pass an empty `str`, so let's change the typing for `group_id` to just `str`.